zaza/charm_lifecycle/deploy.py

```diff
@@ -386,7 +390,8 @@ def deploy(bundle, model, wait=True, model_ctxt=None, force=False,
     run_report.register_event_start('Deploy Bundle')
     with notify_around(NotifyEvents.DEPLOY_BUNDLE, bundle=bundle, model=model,
                        model_ctxt=model_ctxt, force=force):
-        deploy_bundle(bundle, model, model_ctxt=model_ctxt, force=force)
+        deploy_bundle(bundle, model, model_ctxt=model_ctxt, force=force,
+                      trust=trust)
```

Comment:
It would probably be nice to add `trust=trust` to the`notify_around` handler above, so that's recorded with the statistics (like `force` currently is).